</script>
<title>Suvrajit Maji Research</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-"/>
<meta name="viewport" content="width=device-width, initial-scale=1">
<link href="https://github.com/suvrajitm/suvrajitm.github.io/blob/main/myCssStyle.css" rel="stylesheet" type="text/css">


</head>

<link href="myCssStyle.css" rel="stylesheet" type="text/css">
<body>
<div id="container">
<div id="header">
<h1>Research</h1>
<p class="description"></p>
</div>
<div id="wrapper">
<div id="content">
<h2>Research Interests</h2>
<div align="justify"><p>
<b>I. Computational Structural Biophysics and Systems Biology.</b> My current research projects are broadly in the area of structural biophysics and systems biology. I am mainly involved in few projects related to the development of computational methods to analyze and predict the RNA-protein binding interactions, RNA-protein complex structures using crosslinking, sequence and structural datasets.
<br><br><b>II. Computational Structural Biophysics.</b> My primary research area has been computational structural biology and biophysics. In my previous position in Dr. Joachim Frank's lab (04/2015 - 08/2020), I was mainly involved in the development of methods to study the structure and dynamics of macromolecular machines using cryo-EM image datasets. Following are the details of the individual projects:
<br><b>1. Continuum of conformational states from cryo-EM data using manifold learning:</b><br><b>(a)</b> I was working on a method ManifoldEM to obtain a continuum of conformational states of macromolecular machines from cryo-EM data.
Conventional methods using maximum likelihood classification, analyzes the projection images and determines the 3D structures with various conformation.
Hoewever, these methods are able to determine only a limited number of discrete classes.
Our approach aims to determine the continuum of conformational states (landscape) given enough projection images
of the molecular machines (Dashti et al. 2014, Frank et al. 2016). We achieve this task by analyzing the "space of images" employing the methodology known as manifold learning.
<br><b>(b) Propagation of reaction coordinates across angular space:</b>One important step in deriving a consolidated map of occupancies and energy landscape, is to propagate the reaction coordinates, from a known "anchor PD" and this has been done thus far "by hand".
In this work we have developed a method (Accepted to Journal of Chemical Information and Modeling, Special Issue:Frontiers in CryoEM Modeling, Maji et al. 2020), that attempts to solve the propagation problem by using a probabilistic graphical model and feature extraction methods such as optical flow & histogram of oriented gradients.
We illustrate the successful application of the propagation method to a ryanodine receptor and ribosome data.
<br><img alt="" src="http://www.columbia.edu/~sm4073/reswork/ccpropfigures1.jpg" width="475" height="250"></br>

<br><br><b>2. Structure and function of 30S translational initiation complex:</b> Recently, I got involved in an ongoing project in our group using cryo-EM to obtain insight about the conformational rearrangements of the ribosomal subunits during the initiation process.

<br><br><b>3. Characterization of domain motion with coordinate transformation using moments of inertia tensor:</b> I am also working on developing methods to study the domain motion of macromolecular structures. Molecular machines such as the ribosome,
while performing a certain biophysical process (e.g. translation, protein synthesis , etc.), undergo a series of conformational changes often characterized by relative motions of their domains.
These processes can be studied using various structural biology techniques and usually we have conformational snapshots of the molecular machines (pdb structures), obtained from flexible fitting of low-resolution cryo-EM density maps,
from X-ray crystallography, or from molecular dynamics simulation trajectories. The goal is to analyze these snapshots and obtain meaningful information about the motions of the molecular machines.
We have recently developed a comprehensive method (<a href="http://pubs.acs.org/doi/abs/10.1021/acs.jpcb.6b10732" style="TEXT-DECORATION: none">Maji et al. 2017</a>) implemented
on the VMD platform, which can quantify these motions in a general and biologically informative way. Our objective is to characterize the motion of each domain as a
coordinate transformation using moments of inertia tensor (Agirrezabala et al. 2011, 2012). These tools presents a practical way for reproducible segmentation of domains,
and provide a generalized description of their motions
using principal axes of inertia. We can obtain the rotation angle and the native rotation axis for certain domain undergoing motion. Moreover, we can also determine
the component of the rotation around a specified axis, not necessarily the native rotation axis. Using such a toolset we can compare domain motion data from different researchers.
<br><br><img alt="" src="http://www.columbia.edu/~sm4073/reswork/figdmtoc.jpg" width="250" height="175"></br>
<br><b>Quantifying the domain motion of macromolecular machines</b></br>
<br><img alt="" src="http://www.columbia.edu/~sm4073/reswork/SSUBodyRot.gif" width="350" height="175">
<img alt="" src="http://www.columbia.edu/~sm4073/reswork/SUHeadRot.gif" width="350" height="175"></br>
<br><b>Domain Motion of a molecular machine, ribosome. Small Subunit Body rotation (Top). Small Subunit Head rotation (Bottom).</b></br>
<br>Reference:<a href="http://pubs.acs.org/doi/abs/10.1021/acs.jpcb.6b10732" style="TEXT-DECORATION: none"> Quantitative Characterization of Domain Motions in Molecular Machines</a>
<br><a href="http://pubs.acs.org/doi/abs/10.1021/acs.jpcb.6b10732" style="TEXT-DECORATION: none">DOI: 10.1021/acs.jpcb.6b10732 </a></br>
<br><b>Previous Research</b>
<br>My previous research interests were single molecule Biophysics and fluorescence microscopy image analysis. I was involved in few different projects such as single particle tracking of biological molecules and superresolution bioimaging studies.
My main project was on developing approaches using Generative models for inferring biological structures from single molecule super-resolution datasets, which can potentially improve dynamic localization imaging of structural components in cells.
I was also involved in the development of three-dimensional single molecule localization algorithms using fluorescent beads, quantum dots and fluorogen activating peptides (FAPs).
<br>Apart from single molecule imaging projects, I have also worked on understanding co-translational folding of protein, where I computationally studied the effect of codon usage and tRNA concentration on the translation rates for several proteins.<br/>
<br><br>
<h2>Research Summary</h2>
<div align="justify">
<b>I. Computational Biology and Computational Genomics</b>
<li type="none">Transcriptional Gene Regulatory network</li>
<li type="none">Machine Learning</li>
<b>II. Computational Structural Biology and Biophysics</b>
<li type="none">Cryo-EM Image reconstruction of macromolecular structures</li>
<li type="none">Heterogeniety and dynamics of macromolecular machines using Manifold learning and other computational approaches</li> <b>III. Computational Single Molecule Biophysics</b>
<li type="none">Single Molecule Biophysics , Single Particle Tracking, Super-Resolution Imaging</li>
<br><b>IV. General Topics</b></br>
<li type="none">Probabilistic Graphical Models</li>
<li type="none">Generative Models</li>
<li type="none">Manifold Learning</li>
<li type="none">Machine Learning</li>
<li type="none">Biological Image Analysis</li>
<li type="none">Computer Vision</li>
<li type="none">Pure and Applied Mathematics</li>
</div>
</div>

</div>

<div id="navigation">
<h2></h2>
<ul>
<li><a href="http://www.columbia.edu/~sm4073/" style="TEXT-DECORATION: none">Home</a></li>
<li><a href="cv.html" style="text-decoration: none">Curriculum Vitae</a></li>
<li><a style="TEXT-DECORATION: none" href="research.html" class="current">Research</a></li>
<li><a style="TEXT-DECORATION: none" href="publications.html">Publications</a></li>
<li><a style="TEXT-DECORATION: none" href="personal.html">Personal</a></li>
<li><a style="TEXT-DECORATION: none" href="gallery.html">Gallery</a></li>
<li><a style="TEXT-DECORATION: none" href="newsblog.html">News Blog</a></li>
</ul>
</div>
</div>
<div id="extra">
<h2></h2>
<p> </p>
</div>
<div id="footer">
<ul>
<li><a href="http://www.columbia.edu/~sm4073/" style="TEXT-DECORATION: none">Home</a></li>
<li><a href="cv.html" style="text-decoration: none">Curriculum Vitae</a></li>
<li><a style="TEXT-DECORATION: none" href="research.html">Research</a></li>
<li><a style="TEXT-DECORATION: none" href="publications.html">Publications</a></li>
<li><a style="TEXT-DECORATION: none" href="personal.html">Personal</a></li>
<li><a style="TEXT-DECORATION: none" href="gallery.html">Gallery</a></li>
<li><a style="TEXT-DECORATION: none" href="newsblog.html">News Blog</a></li>
</ul>
<p>
<script language="JavaScript">
<!-- Hide JavaScript...
var LastUpdated = document.lastModified;
document.writeln ("last updated " + LastUpdated + " EST");
// End Hiding -->
</script>
</p>
</div>

<div id="container">
<div style="float:left;"><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=190&t=tt&d=ep3zslNnOLbXwk5lIFVOaV02faEbkNoCHwjiJ6c5qw8'></script></div>
<!-- Start: Copyright 2019 TraceMyIP.org Service Code (232631-06242019)- DO NOT MODIFY //-->
<div style="line-height:16px;text-align:right;"><script type="text/javascript" src="//s3.tracemyip.org/tracker/lgUrl.php?stlVar2=1209&amp;rgtype=4684NR-IPIB&amp;pidnVar2=44030&amp;prtVar2=3&amp;scvVar2=12"></script><noscript><div><a href="https://www.tracemyip.org/website-analytics.htm"><img src="//s3.tracemyip.org/tracker/1209/4684NR-IPIB/44030/3/12/ans/" alt="gdpr website statistics" style="border:0px;" /></a></div></noscript></div>
<!-- End: TraceMyIP.org Service Code //-->
</div>
</div>
</body>
</html>
